src/css/variables.less

```diff
@@ -16,6 +16,9 @@
 @link-color: #ffffff;
 @dropdown-bg-color: #ffffff;
 @dropdown-separator-color: #DADCDE;
+@table-header-color:#46494D;
+@table-highlight-color:#3A3C40;
+@table-border-color:#3C3F43;
```

Comment:
Yeah they were totally up for it. Unfortunately Sketch doesnt help us with that, so we need to achieve them by trial and error...